### ISSUE TITLE
Sub categories section is a bit confuse 

### ISSUE THREAD

**Issue by anaximeno** [linuxmint/mintinstall#367]
I initially thought this is some sort of bug when entering a category section where it would still show other categories down below when entering a category, but after a more careful look, I noticed that it was a sub-category section. It seems a bit confusing for me there, wouldn't be better to put that section at the top below the category title (the category title could even be removed from there and put as the title of the window)?

![games](https://github.com/linuxmint/mintinstall/assets/63073056/afb16865-f467-4fec-8811-84bbd12ba317)

![Screenshot from 2023-06-03 03-52-58](https://github.com/linuxmint/mintinstall/assets/63073056/baf695e7-5e5b-456d-a114-b159740d59de)

**Comment by JosephMcc** [comment]
I agree this layout is really weird. Seemed like it was much better in the older layout when these were a sidebar having their own clearly defined area. The way it is now just seems a bit hacked on.

**Comment by marco4mp** [comment]
I agree with the proposal. I think it looks prettier and more intuitive.
I made a quick mockup with Gimp to compare.

![software manager categorías mod vs](https://github.com/linuxmint/mintinstall/assets/52709674/48df8a05-0895-4e7a-84d3-4df57d594f49)

**Comment by anaximeno** [comment]
Still a thing.
